config/custom-environment-variables.yaml

```
@@ -346,3 +346,7 @@ rabbitmq:
     prefetchCount: RABBITMQ_PREFETCH_COUNT
     # Message reprocess limit
     messageReprocessLimit: RABBITMQ_MSG_REPROCESS_LIMIT
+httpd:
+  port: PORT
+  host: HOST
+  uri: URI
```

Would it be better to have an explanation in the comments like [API](https://github.com/screwdriver-cd/screwdriver/blob/cda501eeaab14d8833b983febe8051624c33652a/config/custom-environment-variables.yaml#L49-L55)?
```suggestion
  # Port to listen on
  port: PORT
  # Host to listen on (set to 0.0.0.0 to accept all connections)
  host: HOST
  # Externally routable URI (usually your load balancer or CNAME)
  uri: URI
```